### PR TITLE
fixes, no more !important in _new.scss

### PR DESCRIPTION
**Pull request by wsvincent**

### CODE REVIEW

File: layouts/partials/new_footer.html [hunk 1]
```diff
@@ -22,15 +22,14 @@
...
         <div class="container-flex footer-bottom-icons">
             <ul class="footer-bottom-icon-group">
                 <li><a href="/"><img src="/assets/images/meta-logo-white.svg" alt="Meta Logo"></a></li>
             </ul>
             <ul class="footer-bottom-icon-group">
                 <li class="footer-bottom-icon-spacing"><div class="footer-contact-us"><a href="/contact/">Contact Us</a></div></li>
-                <li class="footer-bottom-icon-spacing"><a href="https://www.facebook.com/metasearch" target="_blank"><i class="fa fa-15x fa-facebook-official" aria-hidden="true"></i></a></li>
-                <li class="footer-bottom-icon-spacing"><a href="https://twitter.com/metasearch" target="_blank"><i class="fa fa-15x fa-twitter" aria-hidden="true"></i></a></li>
+                <li class="footer-bottom-icon-spacing"><a href="https://www.facebook.com/metasearch" target="_blank"><i class="fa fa-2x fa-facebook-official" aria-hidden="true"></i></a></li>
+                <li class="footer-bottom-icon-spacing"><a href="https://twitter.com/metasearch" target="_blank"><i class="fa fa-2x fa-twitter" aria-hidden="true"></i></a></li>
```
Comment (kujenga) [inline layouts/partials/new_footer.html]: These social icons don't seem to be appearing any more

File: layouts/about/single.html [hunk 2]
```diff
@@ -108,9 +108,9 @@
                             <li>William Vincent</li>
                             <li class="about-page-job-title"></li>
                             <li>
-                                <a href="https://github.com/wsvincent" target="_blank"><i class="fa fa-github-square fa-15x" aria-hidden="true"></i></a>
-                                <a href="https://www.linkedin.com/in/wsvincent" target="_blank"><i class="fa fa-linkedin-square fa-15x" aria-hidden="true"></i></a>
-                                <a href="https://wsvincent.com" target="_blank"><i class="fa fa-globe fa-15x"></i></a>
+                                <a href="https://github.com/wsvincent" target="_blank"><i class="fa fa-github-square fa-2x" aria-hidden="true"></i></a>
+                                <a href="https://www.linkedin.com/in/wsvincent" target="_blank"><i class="fa fa-linkedin-square fa-2x" aria-hidden="true"></i></a>
+                                <a href="https://wsvincent.com" target="_blank"><i class="fa fa-globe fa-2x"></i></a>
```
Comment (kujenga) [inline layouts/about/single.html]: These icons are also all much bigger, not sure if that's intended

File: layouts/partials/new_footer.html (hunk 1 above)
Comment (wsvincent) [inline layouts/partials/new_footer.html]: Strange...they seem fine for me across multiple browsers. Will investigate further.

Comment (wsvincent) [inline layouts/partials/new_footer.html]: @kujenga Icon sizing is fixed cleanly now, can't replicate the icons not appearing though you and Jason see that...we can review Monday when I'm in.

File: layouts/partials/header.html [hunk 3]
```diff
@@ -9,6 +9,7 @@
     <meta name="description" content="{{ .Description }}">
     <meta name="uptime-hash" content="3c3323992aa3915a340e7cefc8ecf730f4bbafec" /> <!-- DO NOT MODIFY THIS LINE -->
 
+    <link rel="stylesheet"  href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
     <link rel="stylesheet" type="text/css" href="//cloud.typography.com/6124174/762648/css/fonts.css" />
```
Comment (kujenga) [inline layouts/partials/header.html]: Should the font-awesome have a `type="text/css"` as well?

Comment (wsvincent) [inline layouts/partials/header.html]: Yes, I'll add.

**Comment by jasonbriggs** [comment]
Looks good!

**Pull request merged**